Src/BootCamp.Chapter/HealthCheckSimulator.cs

```diff
@@ -0,0 +1,67 @@
+﻿using BootCamp.Chapter.Logging;
+using System;
+
+namespace BootCamp.Chapter
+{
+    public static class HealthCheckSimulator
+    {
+        public static void Run()
+        {
+            Logger sessionLogger = SetupLogger();
+
+            Person user = new Person(); //We create our test subject
+            ConsolePrompter.WelcomeUser();
+            SetPersonalDetails(user, sessionLogger);
+
+            Console.WriteLine($"BMI equals: {BMICalculator.CalculateBMI(user, sessionLogger)}");
+        }
+
+        private static Logger SetupLogger()
+        {
+            Console.Write("Please select where would you like to log for this session. 0 = Console, 1 = Logfile: ");
+            int logDestination;
+
+            if (int.TryParse(Console.ReadLine(), out int result))
+            {
+                logDestination = result;
+            }
+            else
+            {
+                Console.WriteLine("Invalid value, defaulting to 'Console'.");
+                return new Logger(LogTarget.Console);
+            }
+
+            switch (logDestination)
+            {
+                case 0:
+                    return new Logger(LogTarget.Console);
```

Comment:
you should just create a specific logger, instead of setting logging target.

Reply:
I've got rid of LogTarget completely and instead used ILogger in most of my functions. This way it looks so much nicer!